integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/SlackSteps.java

```diff
@@ -28,6 +28,6 @@ public void slackActionTest(String messageText, int timeoutMinutes) {
                 .atMost(Duration.ofMinutes(timeoutMinutes))
                 .pollInterval(Duration.ofSeconds(1))
                 .untilAsserted(
-                        () -> assertThat(SlackResource.getListOfSlackMessages()).containsAnyOf(messageTextWithoutPlaceholders));
+                        () -> assertThat(SlackResource.getListOfSlackMessages()).anyMatch(msg -> msg.contains(messageTextWithoutPlaceholders)));
```

Comment:
any reason for this change ?
this would change the error message and we would lose the information on what is is checking here

Reply:
The reason is to allow checking also part of the message, previously the whole message has to match the text. 
Could be adjusted to something like:
```suggestion
                        () -> assertThat(SlackResource.getListOfSlackMessages()).as("Searching for message containing text: '%s'", messageTextWithoutPlaceholders).anyMatch(msg -> msg.contains(messageTextWithoutPlaceholders)));
```